lib/grape-swagger/doc_methods/parse_params.rb

```diff
@@ -64,7 +64,9 @@ def document_array_param(value_type)
           if value_type[:is_array]
             if value_type[:documentation].present?
               param_type = value_type[:documentation][:param_type]
-              type = GrapeSwagger::DocMethods::DataType.mapping(value_type[:documentation][:type])
+              if (doc_type = value_type[:documentation][:type])
```

Comment:
Sorry, I don't know how you mean. Instead of above, I could do:

```ruby
if value_type[:documentation][:type]
  doc_type = value_type[:documentation][:type]
  # ...
end
```

Is that better to you?